### PR TITLE
[4.3] com_contact in category_view -> Change the language-string for correct table-head-title

### PR DESCRIPTION
**Pull request by Sieger66**
Pull Request for Issue #41318  .

### Summary of Changes

Change the language-string for correct table-head-title

### Testing Instructions

View in frontend a menue item of Menu_item_type "List Contacts in a Category"
index.php?option=com_contact&view=category
with "Table Headings" is "Show" in the "List Layouts"-Tab of this menue-item.
With this PR:
In the head of the table it is the word "Name" for the column of the names of the contacts 

### Actual result BEFORE applying this Pull Request

Bug:
In the head of the table it is the false word "Title" and not "Name" for the column of names of the contacts.

### Expected result AFTER applying this Pull Request

In the head of the table it is the word "Name" for the column of the names of the contacts

Edit:
Same now in Backend-view for table-title and filter as aspected

### Link to documentations
Please select:
- [X] No documentation changes for docs.joomla.org needed

- [X] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by brianteeman** [comment]
The same issue exists in the admin

image

**Comment by chmst** [comment]
@Sieger66 could you make the corresponding change for the backend? Then it is consistent in frontend and backend

There we have another language key 
here https://github.com/joomla/joomla-cms/blob/4.3-dev/administrator/components/com_contact/forms/contact.xml#L16C40-L16C40
and ehre https://github.com/joomla/joomla-cms/blob/4.3-dev/administrator/components/com_contact/forms/filter_contacts.xml#L111

File: components/com_contact/tmpl/category/default_items.php [hunk 1]
```diff
@@ -88,7 +88,7 @@ class="inputbox" onchange="document.adminForm.submit();"
                 <thead<?php echo $this->params->get('show_headings', '1') ? '' : ' class="visually-hidden"'; ?>>
                     <tr>
                         <th scope="col" id="categorylist_header_title">
-                            <?php echo HTMLHelper::_('grid.sort', 'JGLOBAL_TITLE', 'a.name', $listDirn, $listOrder, null, 'asc', '', 'adminForm'); ?>
+                            <?php echo HTMLHelper::_('grid.sort', 'COM_CONTACT_CONTACT_EMAIL_NAME_LABEL', 'a.name', $listDirn, $listOrder, null, 'asc', '', 'adminForm'); ?>
```
Comment (brianteeman) [inline components/com_contact/tmpl/category/default_items.php]:
```suggestion
                            <?php echo HTMLHelper::_('grid.sort', 'COM_CONTACT_FIELD_NAME_LABEL', 'a.name', $listDirn, $listOrder, null, 'asc', '', 'adminForm'); ?>
```

No visual difference (in english) but this is the same string as used in the admin and the other string is the one used in the contact form

**Review by Sieger66 — COMMENTED** [review]
Request change done.

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on f1b086c5f50ef65fa49adea1bb1975f47dc04388<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41319">issues.joomla.org/tracker/joomla-cms/41319</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on f1b086c5f50ef65fa49adea1bb1975f47dc04388<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41319">issues.joomla.org/tracker/joomla-cms/41319</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41319">issues.joomla.org/tracker/joomla-cms/41319</a>.</sub>

**Comment by obuisard** [comment]
Thank you @Sieger66 !

**Pull request merged**